components/ledger/libs/aws/iam/load.go

```diff
@@ -0,0 +1,37 @@
+package iam
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+func InitFlags(flags *pflag.FlagSet) {
+	flags.String("aws-region", "", "Specify AWS region")
```

Comment:
Only nitpicking, but should we put the flag name in constant ? (if we want to access them outside of the package one day ?)